composer.json

```diff
@@ -25,7 +25,7 @@
     "require": {
         "php": ">=7.0",
         "illuminate/database": "^5.3",
-        "nesbot/carbon": "^1.21",
+        "nesbot/carbon": ">=1.21",
```

Comment:
We don't know if 3.x of Carbon will be a breaking change version. We must disallow 3.x for now

Reply:
Sure